src/libraries/System.Text.Json/tests/Serialization/Object.WriteTests.cs

```diff
@@ -125,5 +126,17 @@ private class Polymorphic
             [JsonPropertyName("p_3")]
             public object P3 => "";
         }
+
+        // https://github.com/dotnet/corefx/issues/40979
```

Comment:
https://github.com/dotnet/runtime/issues/30814 (https://github.com/dotnet/corefx/issues/40979) is closed, can you remove the comment?

Reply:
and also rename the test to reflect the dotnet/runtime issue number.

Reply:
And also rename the test to reflect the dotnet/runtime issue number.

Reply:
The link just points to where the sample test came from. We have such links in a few places where the unit test was there to capture a regression or if it was inspired by a user provided scenario as part of the issue they filed.

Reply:
Yeah I’ll update my PR to update all urls using my app.